features/admin/product/managing_product_variants/product_variant_validation.feature

```diff
@@ -15,7 +15,7 @@ Feature: Product variant validation
         And I specify its code as "VODKA_WYBOROWA_PREMIUM"
         But I do not set its price
         And I try to add it
-        Then I should be notified that prices in all channels must be defined
+        Then I should be notified that prices in "United States" channel must be defined
```

Comment:
I'm just wondering if we shouldn't also display the general flash message 🤔

Reply:
I don't think it's necessary. We have decided to add a general flash message for all forms if there is an error, so I think that should be enough.